extensions/target-specific/chromium/ax-tree/ax-tree.js

```diff
@@ -20,7 +20,19 @@ Loader.OnLoad(function() {
 
     function getManagers() {
         if (managersPromise == null) {
-            managersPromise = DbgObject.global(Chromium.BrowserProcessSyntheticModuleName, "g_ax_tree_id_map").F("Object").array("Pairs").f("second");
+            // g_ax_tree_id_map was replaced by a specialized AXTreeManagerMap. Check for both in case an older
+            // crash dump is inspected.
+            managersPromise = Promise.any([
+                DbgObject.global(Chromium.BrowserProcessSyntheticModuleName, "instance", "base::NoDestructor<ui::AXTreeManagerMap>")
```

Comment:
Yeah this is a bit weird, because it's caching a promised value, which I don't think is super common. But I think going forward, the Promise.any approach is a pretty clean way of guessing where a value in memory is stored - you can just have an array of possible Promise-based locations instead of a bunch of try/catch blocks.